Translation/Extractor/File/FormExtractor.php

```diff
@@ -119,7 +120,24 @@ public function enterNode(Node $node)
                 }
 
                 if ('choices' === $item->key->value) {
+
+                    //Checking for the choice_as_values in the same form item
+                    $choices_as_values = false;
```

Comment:
Yeah but rest of the code is a kind of mix between snake and camel, i'm also going to clean this up :)